docs/network-drivers.md

```diff
@@ -213,6 +215,50 @@ The body of the api call request will look like:
 The brocade driver accepts interface names the same way they would be accepted
 in the console of the switch, ex. ``101/0/10``.
 
+### Juniper QFX5200 driver
+
+#### Switch preparation
+
+1. The driver relies on netconf protocol to manage the switch. 
+It is important switch is configured for responding to netconf connections. 
```

Comment:
Phrasing here. "The switch should be configured for responding to netconf connections" or something like that would be better.

Reply:
addressed in the next commit.